src/Components/CloseEmailButton/index.jsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom"
+
+function CloseEmailButton (){
```

Comment:
Any way you can make this more re-usable? What if it was a generic button with props that allow you to set the link and the text?